pkg/generate/generate_test.go

```diff
@@ -0,0 +1,84 @@
+package generate
+
+import (
+	"context"
+	"errors"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/operator-framework/combo/pkg/combination"
+	testdata "github.com/operator-framework/combo/test/assets/generator"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEvaluate(t *testing.T) {
+	for _, tt := range []struct {
+		name         string
+		file         io.Reader
+		combinations combination.Stream
+		expected     []string
+		err          error
+	}{
+		{
+			name:     "can process a template",
+			file:     strings.NewReader(testdata.EvaluateInput),
```

Comment:
Same here `testdata.EvaluateInput` -> `testdata.GenerateInput` and same for `testdata.EvaluateOutput`